FlowCrypt/Functionality/Services/Local Private Key Services/PassPhraseService.swift

```diff
@@ -89,7 +90,7 @@ final class PassPhraseService: PassPhraseServiceType {
         case .persistent:
             try encryptedStorage.save(passPhrase: passPhrase)
         case .memory:
-            let storedPassPhrases = try encryptedStorage.getPassPhrases(for: "")
+            let storedPassPhrases = try encryptedStorage.getPassPhrases(from: passPhrase.primaryFingerprintOfAssociatedKey)
```

Comment:
either search by `email` and then filter by fingerprint below, or search here by `email+fingerprint` then no need to filter below. If searching by email, expect array. If you decide to write the method to accept email + fingerprint then expect optional one item and return the first found.

Don't use ambiguous input like `for`, `from` because it's not clear if it's email or fingerprint or something else to input. Just `email`, `primaryFingerprint` and similar for clarity.